pytorch_lightning/core/saving.py

```diff
@@ -361,9 +361,10 @@ def save_hparams_to_yaml(config_yaml, hparams: Union[dict, Namespace]) -> None:
     if not os.path.isdir(os.path.dirname(config_yaml)):
         raise RuntimeError(f'Missing folder: {os.path.dirname(config_yaml)}.')
 
-    if Container is not None and isinstance(hparams, Container):
+    if Container is not None and (OmegaConf.is_config(hparams) or 
+                                  OmegaConf.is_config(v) for v in hparams.values() ):
         from omegaconf import OmegaConf
```

Comment:
can you try to import OmegaConf instead of Container at the top? We don't need container any more here

Reply:
sure, but could you elaborate on why we don't need Container here anymore? I'm a bit confused w.r.t that

Reply:
Since you are not using Container, you can use the OmegaConf class as your sentinal for successful import:
```python
if OmegaConf is not None and (OmegaConf.is_config(hparams) or 
```

Reply:
Done.